packages/flutter/lib/src/gestures/events.dart

```diff
@@ -1731,6 +1731,22 @@ abstract class PointerSignalEvent extends PointerEvent {
   });
 }
 
+/// A function used to send information about a PointerEvent back to the engine.
+///
+/// This is used on the web to `preventDefault`.
+typedef RespondPointerEventFn = void Function({required bool allowPlatformDefault});
+
+mixin _RespondablePointerEvent on PointerEvent {
+  /// Responds the current [PointerSignalEvent] in the native embedder.
+  ///
+  /// Used to send information about the current event to the Flutter engine, for
+  /// example, if it should prevent the default behavior of the embedder
+  /// ([allowPlatformDefault] = `true`).
```

Comment:
Lets document the contract for calling this method multiple times. Is that allowed? What happens?